GetARock/GetARock/Screens/SignUp/SetAuthorizationViewController.swift

```diff
@@ -0,0 +1,261 @@
+//
+//  SetAuthorizationViewController.swift
+//  GetARock
+//
+//  Created by 최동권 on 2023/02/10.
+//
+
+import CoreLocation
+import UIKit
+
+final class SetAuthorizationViewController: UIViewController {
+    
+    //MARK: - Property
+    
+    private let locationManager = CLLocationManager()
+    private var preventlocationAuthorizationDeniedIndicator = 0
```

Comment:
p3;

delegate 최초 주입을 분기처리 하기 위함이라서 bool 타입을 쓰면 좀 더 직관적인 네이밍이 가능하지 않을까 싶어요!

```suggestion
        private var isFirstLaunch: Bool = false
```

이런 식으로 하는 것은 어떨까요??

Reply:
0, 1의 값을 사용하니 차라리 Bool값으로 하는게 좋겠네요! 
이 프로퍼티가 첫 권한 changed delegate 함수가 실행되는 것을 막는 Bool값이므로 `isFirstAuthorizationChangeRun`라는 네이밍으로 수정해봤는데 어떨까요? [8178ff3](https://github.com/extreme-rock/GetARock-iOS/pull/117/commits/8178ff32ef18d11cb3de54db5d7ca67d03a4d475)에서 반영해보았습니다.

Reply:
넵 좋습니다~